src/Components/DataEntry/RadioButton.vue

```diff
@@ -0,0 +1,324 @@
+<template>
+	<div class="c-application c-radio-button" :class="[computedClasses, computedRadioColor]">
+		<input
+			:id="computedId"
+			v-model="sync_value"
+			type="radio"
+			:disabled="disabled"
+			v-bind="$attrs"
+			:name="name"
+			v-on="$listeners"
+		/>
+		<label :class="{ disabled, checked: value }" :for="computedId">
+			<Typography :color="computedColor" :type="computedTypographyType">
+				<slot />
+			</Typography>
+		</label>
+	</div>
+</template>
+
+<script>
+import { colorKeys } from '@/src/Elements/Core/Colors';
+import Typography from '@/src/Elements/Core/Typography/Typography';
+import uniqueId from '@/utils/unique-id';
+export const radioColors = ['primary', 'success', 'secondary', 'error'];
+export const radioButtonSizes = ['small', 'medium'];
+
+export default {
+	name: 'RadioButton',
+	inheritAttrs: false,
+	props: {
+		value: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		disabled: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		id: {
+			type: String,
+			default: '',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		name: {
+			type: String,
+			default: '',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		radioColor: {
+			type: String,
+			default: 'primary',
+			validator(value) {
+				return radioColors.indexOf(value) !== -1;
+			},
+		},
+		fontColor: {
+			type: String,
+			default: 'gray500',
+			validator(value) {
+				const isValid = colorKeys.indexOf(value) !== -1;
+				if (!isValid) {
+					console.error(`${value} is not a valid name of the typography color`);
+				}
+				return isValid;
+			},
```

Comment:
```suggestion
import customValidator from '@/utils/custom-validator';

			validator(value) {
				const isValid = colorKeys.indexOf(value) !== -1;
				return customValidator(value, isValid, 'RadioButton', 'fontColor');
			},
```

만들어져있는 `customValidator` 함수를 사용하면 좋겠습니다.

Reply:
customValidator이 있었군요......!